test/e2e/resize_nodes.go

```diff
@@ -330,7 +330,14 @@ func performTemporaryNetworkFailure(c *client.Client, ns, rcName string, replica
 	// and cause it to fail if DNS is absent or broken.
 	// Use the IP address instead.
 
-	iptablesRule := fmt.Sprintf("OUTPUT --destination %s --jump DROP", testContext.CloudConfig.MasterName)
+	destination := testContext.CloudConfig.MasterName
+	if providerIs("aws") {
+		// TODO: Use for all clouds?
+		// TODO: Avoid hard-coding this
+		destination = "127.20.0.9"
```

Comment:
Whoops.  Although more problematically, I now have to figure out how the test is now passing.  The test does verify that the node becomes not-ready, so something is strange here.